crates/polars-time/src/windows/window.rs

```diff
@@ -62,31 +62,16 @@ impl Window {
 
     /// Truncate the given ns timestamp by the window boundary.
     pub fn truncate_ns(&self, t: i64, tz: Option<&Tz>) -> PolarsResult<i64> {
-        let t = self.every.truncate_ns(t, tz)?;
-        self.offset.add_ns(t, tz)
-    }
```

Comment:
i was never a huge fan how of `truncate_*` would both truncate and apply an offset..